code/index.html

```diff
@@ -3,28 +3,131 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
     <title>Project 3</title>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
+    
     <!-- video or image as a header is cool :) -->
+    <header class="hero-image">
+      
+      <div class="inner-hero-header"> 
+        <nav class="navbar">
+          
+          <a href="#" class="logo">Frusto</a>
+          <ul class="nav-menu">
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">About us</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+            </li>
+          
+          </ul>
+          </nav>
+      </div>
+
+      
+      <div class="logo-place">
+        <a href="#" class="logo-mobil">Frusto</a>
+      </div>
+      
+    </header> 
 
+  <main class="main-container">
+    
     <!-- Signup form -->
+    <section class="form-wrapper">
+      <h2>Sign up and start unloading your feelings</h2> 
+      <form id="form" class="form-inputs" action="https://httpbin.org/anything" method="post"> 
+      
+        <div class="textarea">
+          <label for="fname">First name</label>
+          <input id="fname" type="text" name="fname"/>
+        
+          <label for="lname">Last name</label>
+          <input id="lname" type="text" name="lname" />
+
+          <label for="email">email</label>
+          <input id="email" type="email" name="email"/>
+        
+          <label for="password">Password</label>
+          <input id="password" type="password" name="password" required/>
+        </div>
+      
+   
+        <div class="radiobutton">
+          <div>
+            <input id="prof" type="radio" name="occupation" value="prof"/>
+            <label for="prof">young and restless</label>
```

Comment:
The `value="prof"/"student"` and `name="prof"/"student"` combined with the `label="young and restless"/"tired and frustrated"`, also on lines 79 & 80, make it a bit confusing when sending in the form to see what's actually been chosen. Maybe a slightly better choice of "name" and "value" could have been chosen that fit the label better?